pennylane/transforms/make_probs.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Transform to strip return_types and return probabilities instead."""
+# pylint: disable=import-outside-toplevel
+
+import pennylane as qml
+
+from .batch_transform import batch_transform
+
+
+@batch_transform
+def _make_probs(tape, wires=None, post_processing_fn=None):
```

Comment:
for the moment as private function because I dont see the wider use for it.

Reply:
Can you help me to import `batch_transform` in the qinfo module? I tried `from ..transforms.batch_transform import batch_transform` as well as `from pennylane.transforms.batch_transform import batch_transform` but get circular import problems in both cases.

Reply:
Does the following work?

```python
import pennylane as qml

...
@qml.batch_transform
```

Reply:
`E   AttributeError: partially initialized module 'pennylane' has no attribute 'batch_transform' (most likely due to a circular import)' :/

Reply:
Fixed it with romain and david by changing the import order in __init__.py of PL :')